tools/star_rod_idx_to_c.py

```diff
@@ -853,23 +862,24 @@ def transform_symbol_name(symbol):
                 out += INDENT + INDENT  + f".idleAnimations = {read_ptr(d[8], symbol_map)},\n"
                 out += INDENT + INDENT  + f".defenseTable = {read_ptr(d[9], symbol_map)},\n"
                 out += INDENT + INDENT  + f".eventFlags = {read_flags(d[10], 'ActorEventFlags')},\n"
-                out += INDENT + INDENT  + f".elementImmunityFlags = {d[11]},\n" # TODO flags
+                out += INDENT + INDENT  + f".elementImmunityFlags = {read_flags(d[11], 'ElementImmunityFlags')},\n"
+                out += INDENT + INDENT  + f".unk_1C = {d[12]},\n"
                 out += INDENT + "},\n"
 
             out += f"}};\n"
         elif struct["type"] == "Actor":
-            out += f"ActorDesc {struct['name']} = {{\n"
+            out += f"ActorDesc NAMESPACE = {{\n"
 
             d = unpack(">IxBBBhxxIIIBBBBBBBBbbbbbbbb", bytes.read(struct["length"]))
 
             out += INDENT + f".flags = {read_flags(d[0], 'ActorFlags')},\n"
-            out += INDENT + f".type = {d[1]},\n" # TODO enum
+            out += INDENT + f".type = {read_enum(d[1], 'ActorType')},\n"
             out += INDENT + f".level = {d[2]},\n"
             out += INDENT + f".maxHP = {d[3]},\n"
             out += INDENT + f".partCount = ARRAY_COUNT({read_ptr(d[5], symbol_map)}),\n"
             out += INDENT + f".partsData = {read_ptr(d[5], symbol_map)},\n"
             out += INDENT + f".script = {read_ptr(d[6], symbol_map)},\n"
-            out += INDENT + f".statusTable = {read_ptr(d[7], symbol_map)},\n"
+            out += INDENT + f".statusTable = (DictionaryEntry*) {read_ptr(d[7], symbol_map)},\n"
```

Comment:
Thinking it might be better to change the type of `statusTable` rather than do this... I believe there was a discussion in Discord regarding removing DictionaryEntry entirely? What was the outcome of that? I'd be happy to although we might have to modify some code